scripts/centos/openedge.spec

```diff
@@ -24,52 +24,71 @@ This package contains the IoT Edge daemon and CLI tool.
+make openedge
+install -d -m 0755 %{buildroot}/usr/local/bin
+install -m 0755 openedge %{buildroot}/usr/local/bin/
+tar cf - -C example/docker etc | tar xvf - -C %{buildroot}/usr/local
 install -d %{buildroot}%{_unitdir}
 install scripts/debian/openedge.service %{buildroot}%{_unitdir}/openedge.service  
 
 %clean
 rm -rf %{buildroot}
 
 %pre
-# Check for container runtime
+
+%post
 if ! /usr/bin/getent group docker >/dev/null; then
     echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
     echo ""
-    echo " ERROR: No container runtime detected."
+    echo " WARNING: docker is uninstalled!"
+    echo ""
+    echo " If you need run openedge in docker container mode, please install docker first:"
     echo ""
-    echo " Please install a container runtime and run this install again."
+    echo " 'curl -sSL https://get.docker.com | sh'"
+    echo " 'systemctl enable docker'"
+    echo " 'systemctl restart docker'"
     echo ""
     echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
-
-    exit 1
+    echo ""
+    echo ""
+fi
+if [ ! -x /usr/bin/systemctl ]; then
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+    echo ""
+    echo " WARNING: systemd is uninstalled!"
+    echo ""
+    echo " OpenEdge need be supervised by daemon tools, such as systemd or supervisor."
+    echo " Otherwise it will exit and can't restart during the master OTA. If you only "
+    echo " want to run openedge in the foregroud, use the following command: "
+    echo ""
+    echo " 'openedge start'"
+    echo ""
+    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+    echo ""
+    echo ""
 fi
-
-exit 0
-
-%post
 echo "==============================================================================="
 echo ""
 echo "                              OpenEdge"
+echo "  OpenEdge is started and supervised by systemd. Use the following commands "
+echo "  to start, restart or stop openedge:"
 echo ""
-echo "  IMPORTANT: Please download your configuration file from your cloud management"
-echo "  suits, then put them in"
-echo "    /usr/local/"
-echo ""
-echo "  You will need to restart the 'openedge' service for these changes to take"
-echo "  effect."
-echo ""
-echo "  To restart the 'openedge' service, use:"
-echo ""
+echo "    'sudo systemctl start openedge'"
 echo "    'sudo systemctl restart openedge'"
+echo "    'sudo systemctl stop openedge'"
 echo ""
-echo "    - OR -"
+echo "  About configurations or design of OpenEdge, please visit "
 echo ""
-echo "    sudo /etc/init.d/openedge restart"
+echo "  https://openedge.tech/en/docs/overview/OpenEdge-design for help."
 echo ""
-echo "  These commands may need to be run with sudo depending on your environment."
+echo "  OpenEdge is running in docker container mode defaultly. Meantime OpenEdge "
+echo "  support native process mode. "
+echo ""
```

Comment:
echo "  OpenEdge is running in docker container mode by default. And OpenEdge"
echo "  also supports native process mode. "